katago_server/trainings/services/bayesian_elo.py

```diff
@@ -1,6 +1,7 @@
 from math import exp, log, sqrt
 import logging
 
+import numpy as np
```

Comment:
Is it needed ?

Reply:
Yes, it fixed a bug from earlier commit.